test/common/testlib.py

```diff
@@ -789,6 +790,23 @@ def enable_preload(self, package, *pages):
         self.write_file(path, '{ "preload": [%s]}' % ', '.join('"{0}"'.format(page) for page in pages))
 
     def setUp(self, restrict=True):
+
+        if os.getenv("MACHINE"):
+            # apply env variable together if MACHINE envvar is set
+            opts.address = os.getenv("MACHINE")
+            if self.is_nondestructive():
+                pass
+            elif os.getenv("DESTRUCTIVE") and not self.is_nondestructive():
+                print("Run destructive test, be careful, may lead to upredictable state of machine")
+            else:
+                raise unittest.SkipTest("Skip destructive test by default")
+            if os.getenv("BROWSER"):
+                opts.browser = os.getenv("BROWSER")
+            if os.getenv("TRACE"):
+                opts.trace = True
+            if os.getenv("SIT"):
+                opts.trace = True
```

Comment:
This seems wrong, shouldn't that be `opts.sit` in the secend case?

Reply:
You are right, fixed. Thanks!